options.c

```diff
@@ -52,6 +52,7 @@ int init_options(int opt_cnt, char *opt_arr[], struct tool_options *options)
 	static struct option long_opts[] = {
 		{"peer", no_argument, NULL, 'u'}, /* UFS device */
 		{"local", no_argument, NULL, 'l'}, /* UFS host*/
+		{"static", no_argument, NULL, 's'},
```

Comment:
I try to alloc below code
{"static", no_argument, opt->set_type, 1},
but, it makes "initializer element is not constant" error
so, i make simple option